bundles/aero.minova.rcp.rcp/src/aero/minova/rcp/rcp/parts/WFCDetailPart.java

```diff
@@ -527,7 +527,7 @@ private void layoutSectionClient(HeadOrPageOrGridWrapper headOrPageOrGrid, Secti
 		// Setzen der TabListe für die einzelnen Sections.
 		clientComposite.setTabList(getTabListForSectionComposite(mSection, clientComposite));
 		// Setzen der TabListe der Sections im Part.
-		clientComposite.getParent().setTabList(getTabListForSection(clientComposite.getParent()));
```

Comment:
Das clientComposite.getParent() ist etwas umständlich, weil du ja weißt dass das parent section ist, wurde ja in Zeile 518 erst erstellt.

Reply:
korrigiere ich gleich